### PR TITLE
Added django 4 to CI

### LINKED ISSUE #579: Add django 4 to CI
**Problem Statement**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by WisdomPill**
Fixes #579 

Added support for django 4.0

### CODE REVIEW

**Review by hartwork — COMMENTED** [review]
Hi @WisdomPill , four things I noticed:
- Adding support for Django 4 seems uncontroversial, dropping 2.2 prior to EOL could be considered controversial by some; I'd split those in two pull requests personally
- The current title of the pull request does not mention dropping Django 2.2; my vote for adding that
- Django 3.1 is EOL even before Django 2.2 (see image below) but it is not being dropped; why not?
- Dropping of Django 2.2 seems to be incomplete, e.g. I still see a line `2.2: dj22` in `setup.cfg`, maybe more.

Just my 2 cents.

Django releases

**Comment by hartwork** [comment]
> * Dropping of Django 2.2 seems to be incomplete, e.g. I still see a line `2.2: dj22` in `setup.cfg`, maybe more.

PS: @WisdomPill there is indeed one more: `README.rst` still says Django `2.2+`.

**Comment by WisdomPill** [comment]
my mistake, dropping 2.2 is a mistake the docs were saying that support for django 1.11 has dropped in april 2020

**Review by hartwork — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -40,12 +40,19 @@ jobs:
             redis-version: 'latest'
             python-version: '3.10'
 
-          # TODO: re-enable with https://github.com/jazzband/django-redis/issues/569
           # pre-release Django and redis
           - django-version: 'main'
             redis-version: 'master'
             python-version: '3.10'
 
+        # exclude python 3.6 and 3.7 for django 4.x
```
Comment (hartwork) [inline .github/workflows/ci.yml:48]: Maybe a rationale on why those are excluded would make this perfect…

Comment (WisdomPill) [reply .github/workflows/ci.yml:48]: They are not supported, maybe they are using new syntax or are looking forward to it. I will write something

Comment (hartwork) [reply .github/workflows/ci.yml:48]: Thanks!

**Comment by WisdomPill** [comment]
thanks @hartwork !

**Pull request merged**